src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -45,6 +45,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)RemoveUnreachableCode\RemoveUnreachableCodeHelpers.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)SimplifyBooleanExpression\CSharpSimplifyConditionalDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)SimplifyInterpolation\CSharpSimplifyInterpolationDiagnosticAnalyzer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)SimplifyLinqExpressions\CSharpSimplifyLinqExpressionsDiagnosticAnalyzer.cs" />
```

Comment:
make Singular.  i.e. it's not `CSharpSimplifyInterpolations` it's `CSharpSimplifyInterpolation`.  The same shoudl hold here.